test/unit/server_async_transaction.test.js

```diff
@@ -0,0 +1,122 @@
+            .then(function () {
+                done();
+            })
+            .catch(function (err) {
+                done(err);
+            });
+    });
+    it("sends an async transaction and gets a PENDING response", function (done) {
+        const response = {
+            tx_status: "PENDING",
+            hash: "db2c69a07be57eb5baefbfbb72b95c7c20d2c4d6f2a0e84e7c27dd0359055a2f",
+        };
+
+        this.axiosMock
+            .expects("post")
+            .withArgs(
+                "https://horizon-live.stellar.org:1337/transactions_async",
+                `tx=${this.blob}`,
+            )
+            .returns(Promise.resolve({ data: response }));
+
+        this.server
+            .submitAsyncTransaction(this.transaction, { skipMemoRequiredCheck: true })
+            .then(function (res) {
+                expect(res.tx_status).to.equal("PENDING")
+                expect(res.hash).to.equal("db2c69a07be57eb5baefbfbb72b95c7c20d2c4d6f2a0e84e7c27dd0359055a2f")
+                done();
+            })
+            .catch(function (err) {
+                done(err);
+            });
+    });
+    it("sends an async transaction and gets a Problem response", function (done) {
+        const response = {
+            type:   "transaction_submission_exception",
+            title:  "Transaction Submission Exception",
+            status: 500,
+            detail: "Received exception from stellar-core." +
+                "The `extras.error` field on this response contains further " +
+                "details.  Descriptions of each code can be found at: " +
+                "https://developers.stellar.org/api/errors/http-status-codes/horizon-specific/transaction-submission-async/transaction_submission_exception",
+            extras: {
+                envelope_xdr: "AAAAAIUAEW3jQt3+fbT6nCASA1/8RWdp9fJ2woxqPHZPQUH/AAAAZAEH/OgAAAAgAAAAAQAAAAAAAAAAAAAAAFyIDD0AAAAAAAAAAQAAAAAAAAADAAAAAAAAAAFCQVQAAAAAAEZK09vHmzOmEMoVWYtbbZcKv3ZOoo06ckzbhyDIFKfhAAAAAAExLQAAAAABAAAAAgAAAAAAAAAAAAAAAAAAAAFPQUH/AAAAQHk3Igj+JXqggsJBFl4mrzgACqxWpx87psxu5UHnSskbwRjHZz89NycCZmJL4gN5WN7twm+wK371K9XcRNDiBwQ=",
+                error: "There was an exception when submitting this transaction."
+            }
+        };
+
+        this.axiosMock
+            .expects("post")
+            .withArgs(
+                "https://horizon-live.stellar.org:1337/transactions_async",
+                `tx=${this.blob}`,
+            )
+            .returns(Promise.resolve({ data: response }));
+
+        this.server
+            .submitAsyncTransaction(this.transaction, { skipMemoRequiredCheck: true })
+            .then(function (res) {
+                expect(res.type).to.equal("transaction_submission_exception")
+                expect(res.title).to.equal("Transaction Submission Exception")
+                expect(res.status).to.equal(500)
+                expect(res.detail).to.equal("Received exception from stellar-core." +
```

Comment:
can you do `.to.equal(response)` to avoid the copy-paste differentiating over time?